src/interfaces/cli/AGENTS.md

```
@@ -0,0 +1,111 @@
+
+---
+
+## OVERVIEW
+
+CLI de memoria usando Typer. Comandos: save, search, list, get, delete. Expuesto como `memory` via pyproject.toml console_scripts.
+
+---
+
+## STRUCTURE
+
+```
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)